pkgs/development/libraries/lensfun/default.nix

```diff
@@ -21,17 +21,20 @@ stdenv.mkDerivation {
     owner = "lensfun";
     repo = "lensfun";
     rev = "v${version}";
-    sha256 = "0isli0arns8bmxqpbr1jnbnqh5wvspixdi51adm671f9ngng7x5r";
+    sha256 = "0ixf0f7qv0mc7zrw9w1sb60w833g4rqrfj8cjxwzv2vimqcksccz";
   };
 
   # replace database with a more recent snapshot
   prePatch = ''
-    rm -R ./data/db
-    cp -R ${lensfunDatabase}/data/db ./data
+    rm -R data/db
+    python3 ${lensfunDatabase}/tools/lensfun_convert_db_v2_to_v1.py $TMPDIR ${lensfunDatabase}/data/db
```

Comment:
Can we add a little note here (or alongside the comment in L27 why we need to convert back to v1, and starting with which lensfun version we won't have to anymore?